scaler/io/utility.py

```diff
@@ -20,3 +20,15 @@ def deserialize(data: bytes) -> Optional[Message]:
 def serialize(message: Message) -> bytes:
     payload = _message.Message(**{PROTOCOL.inverse[type(message)]: message.get_message()})
     return payload.to_bytes()
+
+
+def chunk_to_list_of_bytes(data: bytes) -> List[bytes]:
+    # TODO: change to list of memoryview when capnp can support memoryview
+    return [data[i : i + CAPNP_DATA_SIZE_LIMIT] for i in range(0, len(data), CAPNP_DATA_SIZE_LIMIT)]
+
+
+def concat_list_of_bytes(data: List[bytes]) -> bytes:
+    one_object_bytes = bytearray()
+    for chunk in data:
+        one_object_bytes.extend(chunk)
+    return one_object_bytes
```

Comment:
Looks like it could be implemented with `b"".join([..])`:

```python
In [4]: b"".join(b"hello" for i in range(0, 4))
Out[4]: b'hellohellohellohello'
```